File: app/src/main/java/net/radityalabs/contactapp/presentation/ui/foundation/Dialogs.java
package net.radityalabs.contactapp.presentation.ui.foundation;

/**
 * Created by radityagumay on 4/12/17.
 */

public interface Dialogs {
    void showLoading();

    void showLoading(String message);

    void showNoCancelableLoading();

    void showNoCancelableLoading(String message);

    void hideLoading();
}

File: app/src/main/java/net/radityalabs/contactapp/presentation/ui/foundation/Dialogs.kt
package net.radityalabs.contactapp.presentation.ui.foundation

/**
 * Created by radityagumay on 4/12/17.
 */

interface Dialogs {
    fun showLoading()

    fun showLoading(message: String)

    fun showNoCancelableLoading()

    fun showNoCancelableLoading(message: String)

    fun hideLoading()
}
